src/SArray.jl

```diff
@@ -1,72 +1,70 @@
 """
-    SArray{Size, T, L}(x::NTuple{L, T})
-    SArray{Size, T, L}(x1, x2, x3, ...)
+    SArray{S, T, L}(x::NTuple{L, T})
+    SArray{S, T, L}(x1, x2, x3, ...)
```

Comment:
This I always find a bit weird. In some sense, we are documenting the type *and* the constructors. The one without `T` and `L` is documented below.

The `L` is necessary to know about whenever you want to specify it as an element type of an array or a member of a struct, so this was my effort to get the user to find out about it's existence.

Though now I notice that I forgot the `N`!